sites/360.canonical.com.yaml

```diff
@@ -17,11 +17,6 @@ env:
       key: database_url
       name: threesixty-database-url
 
-  - name: SECRET_KEY
-    secretKeyRef:
-      key: secret_key
-      name: threesixty
```

Comment:
@tbille You need to be careful with this one, k8s doesn't allow numbers for deployments but I can see that the secret name you created in our cluster is 360-canonical-com, it should be threesixty-canonical-com.

Reply:
ooooh good catch